src/main/java/redis/embedded/Cluster.java

```diff
@@ -0,0 +1,13 @@
+package redis.embedded;
+
+/**
+ * Created by dragan on 17.07.15.
+ */
+public interface Cluster extends Redis{
```

Comment:
I agree, but names are always problematic)... 
My option is rename RedisCluster to "SentinelReplicationRedisCluster" or "RedisReplicationModel" (it should has "Replication" part as association with documentation) and "RealRedisCluster" to "RedisCluster" (as far as it is what it is)

What do you think?